src/js/views/patients/worklist/worklist-list.scss

```diff
@@ -28,7 +28,13 @@
   display: inline-block;
   margin-right: 16px;
   vertical-align: middle;
-  width: 16px;
+}
+
+.worklist-list__attachment {
+  color: #999;
+  display: inline-block;
+  margin-left: 8px;
+  vertical-align: middle;
```

Comment:
This `.worklist-list__attachment` and `.worklist-list__details` above it have almost identical properties. 

Maybe we could combine them and make use of `.u-margin--l-8` and `.u-margin--r-16` for the `margin-right`/`margin-left` differences?

Reply:
yep I think that's ideal.  perhaps the class can be something more generic.  `__display-icon` or something?

Reply:
That works well. Updated in latest push.